### ISSUE TITLE
Graphics anomaly on compositor2 branch in radio canvas

### ISSUE THREAD

**Issue by NikolaiVChr** [NikolaiVChr/f16#383]
![Gpx artifact on radio canvas](https://cdn.discordapp.com/attachments/496121096492417025/907052059075940412/Screen_Shot_2021-11-07_at_23.31.35.jpg)

Reported by SammySkyCraft

**Comment by NikolaiVChr** [comment]
Related to issue #187

**Comment by NikolaiVChr** [comment]
When starting the 2020.4.0 (win nightly 23th Dec 2021) sim up, this issue is not present for me.
When changing shadow quality, and pressing 'close' on the custom rendering dialog, all 6 display have issues.
No OpenGL errors in console.

The displays all have the lcd effect applied to them, except HUD. They are all canvas.

It seems to be a canvas problem, as when I remove LCD effect on freq display, its still broken.

![fgfs-20220105203847](https://user-images.githubusercontent.com/3764194/148287697-0eeb4f01-bd94-4fbc-8fe7-1ba55ac7f866.png)

**Comment by NikolaiVChr** [comment]
Conclusion: @SammySkycrafts I think this is bug in FG.

**Comment by legoboyvdlp** [comment]
Yes, it is known that after you change some graphics options, Canvas stops working. I'd suggest you do **not** use 2020.4 right now.

Since a while, when I change the shadow setting, FlightGear stops rendering (doesn't freeze - just doesn't render).

**Comment by NikolaiVChr** [comment]
Another person had this issue. I am reopening it, so we pay attention to it.

**Comment by NikolaiVChr** [comment]
Likely fixed with b308d6df86086375491bd6786b6fa3a50c2965b3
